azure/terraform/app.tf

```
@@ -0,0 +1,19 @@
+resource "azuread_application_registration" "ternary" {
+  display_name     = "Ternary2"
```

```suggestion
  display_name     = "Ternary"
```

I assume this was a temp change to avoid conflict with the existing registered application while developing this TF module?

---

ah yep, good eye - fixed